src/bp-activity/actions/dislike.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Activity: Dislike action
+ *
+ * @package BuddyPress
+ * @subpackage ActivityActions
+ * @since 14.0.0
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Dislike an activity.
+ *
+ * @since 14.0.0
+ *
+ * @return bool False on failure.
+ */
+function bp_activity_dislike_action() {
+	if ( ! is_user_logged_in() || ! bp_is_activity_component() || ! bp_is_current_action( 'dislike' ) ) {
+		return false;
+	}
+
+	// Check the nonce.
+	check_admin_referer( 'bp_activity_like' );
+
+	$feedback        = __( 'Ouch something went wrong when trying to dislike this activity.', 'buddypress' );
+	$feedback_type   = 'error';
+	$activities      = bp_activity_get_specific(
+		array(
+			'activity_ids'     => bp_action_variable( 0 ),
+			'display_comments' => 'threaded',
```

Comment:
Why is this necessary?

Reply:
To validate the "like" exists, we'll need to deprecated `display_comments` in favor of `display_reactions`. This is necessary because likes as comments are Activity children.